docs/subsystems/notifications.md

```diff
@@ -124,8 +124,8 @@ as follows:
   `push_notifications.py` code that actually sends the
   notification. This logic is somewhat complicated by having to track
   the number of unread push notifications to display on the mobile
-  apps' badges, as well as using the [Mobile Push Notifications
-  Service](../production/mobile-push-notifications.md) for self-hosted
+  apps' badges, as well as using the [mobile push notifications
+  service](../production/mobile-push-notifications.md) for self-hosted
```

Comment:
I don't want to rename "Zulip Cloud" and "Mobile Push Notifications Service", at least without discussion.

Reply:
I decided this was OK as an informal use case, but the registration script should match https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html, which is inconsistent, so I left that as-is to avoid churn until we decide how we want to standardize that.

Reply:
I don’t think there was a proposal to rename Zulip Cloud, which is clearly a proper noun. If we want to make the notifications service into a proper noun, it seems to me it should have Zulip in the name?

By the way, we’re inconsistent about “notification service” vs. “notifications service”.